Fabric.Authorization.API/Infrastructure/Middleware/AngularMiddleware.cs

```diff
@@ -37,12 +37,16 @@ public async Task Invoke(HttpContext context)
                 var fullPath = Path.Combine(_hostingEnvironment.WebRootPath, AccessControl.Path, AccessControl.Index);
                 _indexContent = File.ReadAllText(fullPath);
 
-                var discoveryServiceSettings = _appConfiguration.DiscoveryServiceSettings;
+                var discoveryServiceSettings = _appConfiguration.DiscoveryServiceSettings;
+                var identityServerSettings = _appConfiguration.IdentityServerConfidentialClientSettings;

```

Comment:
I see you have added some of the things I am working on.  Probably needed for the move.  Ill keep this in mind during merge time.

Reply:
Yup - I tried to use the same pattern you used so it would make the merge easier...